idaes/generic_models/unit_models/heat_exchanger.py

```diff
@@ -277,66 +309,67 @@ def build(self):
                 "transfer calculation for cross flow.",
             )
             f = self.crossflow_factor
-        ########################################################################
-        # Add control volumes                                                  #
-        ########################################################################
-        _make_heater_control_volume(
-            self,
-            "side_1",
-            config.hot_side_config,
-            dynamic=config.dynamic,
-            has_holdup=config.has_holdup,
-        )
-        _make_heater_control_volume(
-            self,
-            "side_2",
-            config.cold_side_config,
-            dynamic=config.dynamic,
-            has_holdup=config.has_holdup,
-        )
-        # Add named references to side_1 and side_2, side 1 and 2 maintain
-        # backward compatability and are names the user doesn't need to worry
-        # about. The sign convention for duty is heat from side 1 to side 2 is
-        # positive
-        add_object_reference(self, config.hot_side_name, self.side_1)
-        add_object_reference(self, config.cold_side_name, self.side_2)
-
-        # Add convienient references to heat duty.
-        self.heat_duty = Reference(self.side_2.heat)
-        # Need to do this as Reference does not have units (Pyomo fixing this)
-        q = self.side_2.heat
+
+        self.heat_duty = Reference(cold_side.heat)
+        q = self.heat_duty
```

Comment:
Can we avoid the alias for `self.heat_duty`? Might be easier to read the code instead of just `q`.

Reply:
Agree, will fix.